pkg/bootloader/grub.go

```diff
@@ -402,8 +393,8 @@ func (g *Grub) SetDefaultEntry(partMountPoint, imgMountPoint, defaultEntry strin
 }
 
 // Install installs grub into the device, copy the config file and add any extra TTY to grub
-func (g *Grub) Install(rootDir, bootDir, stateLabel string) (err error) {
-	err = g.InstallEFI(rootDir, bootDir, constants.EfiDir, stateLabel)
+func (g *Grub) Install(rootDir, bootDir, deviceLabel string) (err error) {
+	err = g.InstallEFI(rootDir, bootDir, constants.EfiDir, deviceLabel)
```

Comment:
We can finally get rid of this `constants.EfiDir` parameter and use `bootDir` instead. Keeping them as separate parameters at `InstallEFI` method probably makes sense, even if with the current setup they are the same value.

Probably there are more  `constants.EfiDir` references that now can be parametrized as the bootDIr.

Reply:
Yep, I left it in not to make any breaking changes in the interface. We could consider this in a follow up PR!